quickemu

```diff
@@ -0,0 +1,47 @@
+---
+tags: [ virtualization ]
+---
+# To run a vm
+quickemu (--vm <path-to-vm.conf>)
```

Comment:
```suggestion
quickemu --vm <path-to-vm.conf>
```

Optional change: unexpected `(`: braces can be removed.

```suggestion
quickemu (--vm <path/to/vm.conf>)
```
Optional change: `path/to/...` format can be used.

```suggestion
quickemu --vm <path/to/vm.conf>
```

Optional change: fixes can be applied together.